packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js

```diff
@@ -123,17 +123,17 @@ export default class DkimAutoConfigurator {
     );
   }
 
-  writeError(msg, args) {
-    this.writeMessage(msg, 'error', args);
+  writeError(message, params) {
+    this.writeMessage(message, 'error', params);
   }
 
-  writeSuccess(msg, args) {
-    this.writeMessage(msg, 'success', args);
+  writeSuccess(message, params) {
+    this.writeMessage(message, 'success', params);
   }
 
-  writeMessage(msg, type, args) {
-    const message = this.services.$translate.instant(msg, args);
-    this.services.$scope.setMessage(message, {
+  writeMessage(message, type, params) {
+    const translation = this.services.$translate.instant(message, params);
+    this.services.$scope.setMessage(translation, {
       type,
     });
```

Comment:
`writeError` -> `setErrorMesssage`, `writeSuccess` -> `SetSuccessMessage`
I dont see the need for `writeMessage` wrapper.

Reply:
DKIM has a lot of duplicated code, so to ease improvements or a potential refactoring, the code is kept quite similar to exchange part (https://github.com/ovh/manager/blob/feat/MANAGER-12167/packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js#L68)